modules/components/partials/fields/caches/redpanda.adoc

```diff
@@ -1,7 +1,16 @@
 // This content is autogenerated. Do not edit manually. To override descriptions, use the doc-tools CLI with the --overrides option: https://redpandadata.atlassian.net/wiki/spaces/DOC/pages/1247543314/Generate+reference+docs+for+Redpanda+Connect
+// tag::single-source[]
 
 == Fields
 
+=== `allow_auto_topic_creation`
```

Comment:
Looks like this isn't present in the `Advanced` section, where all parameters should be listed. Not sure why that's the case...